src/components/Verse/VerseText.module.scss

```diff
@@ -5,7 +5,8 @@
 
 .verseText {
   display: flex;
-  align-items: center;
+  // flex-start for when the inline wbw translation/transliteration is enabled and its content is wrapped across multiple lines.
```

Comment:
nit: let's either rename "wbw" to "word by word", or change the "word by word" comment in `InlineWordByWord.module.scss` to "wbw". Let's also put a link for the case you're talking about. Maybe https://quran.com/2:74, right?